src/resources/OrganizationDefinitions/OrganizationDefinitions.ts

```diff
@@ -0,0 +1,15 @@
+import Resource from '../Resource';
+import API from '../../APICore';
+import {OrganizationDefinitionModel} from './OrganizationDefinitionInterfaces';
+
+export default class OrganizationDefinitions extends Resource {
```

Comment:
I think it would make sense if this one was fronted through the [`Organization`](https://github.com/coveo/platform-client/blob/master/src/resources/Organizations/Organization.ts) resource so you would do `platform.organization.definitions.get()`. What do you think?

Reply:
Aight j'peux le bouger